serde-api/src/main/java/io/micronaut/serde/Serializer.java

```diff
@@ -120,5 +122,23 @@ default boolean hasView(Class<?>... views) {
         <B, P> SerializationReference<B, P> resolveReference(
                 @NonNull SerializationReference<B, P> reference
         );
+
+        /**
+         * Get the {@link SerdeConfiguration} for this context.
+         *
+         * @return The {@link SerdeConfiguration}
+         * @since 2.6.0
+         */
+        @NonNull
+        SerdeConfiguration getSerdeConfiguration();
```

Comment:
Micronaut Data implements the context so this will fail the compilation. Maybe we can introduce a new interface for this configuration or move it to `SerializerLocator`?.

Reply:
SerializerLocator wouldnt help because EncoderContext extends SerializerLocator.

Maybe I can add a DelegatingEncoderContext that data can extend?